libs/MotionKit/include/MotionKit.h

```diff
@@ -16,8 +16,12 @@ class MotionKit
 {
   public:
 	MotionKit(interface::Motor &motor_left, interface::Motor &motor_right, IMUKit &imu_kit,
-			  interface::EventLoop &event_loop)
-		: _motor_left(motor_left), _motor_right(motor_right), _imukit(imu_kit), _event_loop(event_loop) {};
+			  interface::EventLoop &event_loop, interface::Timeout &timeout)
+		: _motor_left(motor_left),
+		  _motor_right(motor_right),
+		  _imukit(imu_kit),
+		  _event_loop(event_loop),
+		  _timeout(timeout) {};
```

Comment:
having a `;` is an error

```suggestion
		  _timeout(timeout) {}
```